src/sentry/tasks/merge.py

```diff
@@ -228,6 +229,50 @@ def rehash_group_events(group_id, transaction_id=None, **kwargs):
     delete_group.delay(group.id)
 
 
+def decrement_new_issue_count(group):
+    """
+    Updates the new issue/groups count
+
+    Should be called with the old group from merge_group.
+    That will be deleted, and so the count is decremented by 1.
+    """
+    from sentry.models import GroupRelease, Release, ReleaseProject, ReleaseProjectEnvironment
+    release_version = group.get_first_release()
+    if release_version is None:
+        return
+    try:
+        release = Release.objects.get(version=release_version)
```

Comment:
You're going to need to include the `organization_id` in here both to restrict this to the organization of the group being merged (right now this would update every version that matches, so `1.0.0` in every organization, for instance) as well as to utilize the available indexes:

```
    "sentry_release_pkey" PRIMARY KEY, btree (id)
    "sentry_release_organization_id_f0a7ec9ba96de76" UNIQUE, btree (organization_id, version)
    "sentry_release_organization_id" btree (organization_id)
    "sentry_release_organization_id_b3241759a1649e" btree (organization_id, (COALESCE(date_released, date_added)))
    "sentry_release_primary_owner_id" btree (owner_id)
```

Reply:
D: good catch! I didn't realize that!!!